14/index.php

```diff
@@ -0,0 +1,174 @@
+  ]
+]);
+?>
+
+<h2>Announcing the release of Keyman 14.0 Beta</h2>
+<p class="red">30 November 2020</p>
+<p>We are pleased to announce that Keyman 14.0 is now in beta!</p>
+<br>
+
+<h3>What's New?</h3>
+<ul>
+  <li>
+    Simpler and Smoother Keyboard Seach<br>
+    <img alt='Search for Khmer Keyboard' src='keyboard_search_khmer.png'>
+  </li>
+  <li>
+    Localizable UI through <a href="https://translate.keyman.com">translate.keyman.com</a>.
+  </li>
+  <li>
+    Mobile apps download and install keyboard packages from keyman.com
+  </li>
+  <li>
+    Consolidated crash reporting to <a href="https://sentry.keyman.com">sentry.keyman.com</a>
+  </li>
+</ul>
+<br>
+
+<h3>What's Next?</h3>
+<ul>
+  <li>
+    Check the <a href="https://blog.keyman.com/2020/03/keyman-roadmap-march-2020/">Keyman Roadmap</a> for upcoming features.
+  </li>
+</ul>
+<br>
+
+<h3>14.0 Beta Feedback</h3>
+<ul>
+  <li>Please send feedback about Keyman 14 to the
+    <a href="https://community.software.sil.org/t/keyman-14-beta-feedback/4115">Keyman Community site</a> or submit bugs and feature requests to our
+    <a href="https://github.com/keymanapp/keyman/issues/new/choose">Issue Tracker</a></li>
+</ul>
+<br>
+
+<h1 class='red underline'>User Software</h1>
+
+<?php
+downloadSection('Keyman for Windows 14',   'windows',     'keyman-$version.exe', 'beta');
+?>
+
+<h3>What's New in Keyman for Windows 14?</h3>
+
+<ul>
+  <li>Renamed from <strong class="red">Keyman Desktop</strong> to <strong class="red">Keyman for Windows</strong></li>
+  <li>Handles major version upgrades from 10 onward</li>
+  <li>Add GUI for setting configuration options</li>
+</ul>
+
+
+
+<?php
+downloadSection('Keyman 14 for macOS',   'mac',     'keyman-$version.dmg', 'beta');
+?>
+
+<h3>What's New in Keyman 14 for macOS?</h3>
+
+<ul>
+  <li>Added icon for Keyman.app</li>
+</ul>
+
+
+
+<h2 id='linux' class='red underline'>Keyman 14 for Linux</h2>
+
+<h3>What's New in Keyman 14 for Linux?</h3>
+
+<ul>
+  <li>Open a .kmp file with Keyman Config</li>
+  <li>Now supports Ubuntu 20.10 (Groovy)</li>
+  <li>Improved UI</li>
+</ul>
+
+<li>Ubuntu, Wasta-Linux: Keyman for Linux can be installed via launchpad:</li>
+<blockquote><pre class='language-bash code'><code>sudo add-apt-repository ppa:keymanapp/keyman-nightly
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install keyman onboard</code></pre></blockquote>
+
+
+<?php
+downloadSection('Keyman for Android 14', 'android', 'keyman-$version.apk', 'beta');
+?>
+
+<p>You can also <a href=<?= $playstore_signup_link ?>>sign up</a> to access pre-release versions through Google Play.</p>
+<?= $playstoreTable ?>
+
+<h3>What's new in Keyman for Android 14?</h3>
+
+<ul>
+  <li>
+    Select a language during keyboard package installation<br>
+    <img alt='Select language during keyboard installation' src='select_language.png'>
+  </li>
+  <li>Consolidated install menus for installing keyboards</li>
+  <li>Add system globe action to show system keyboards</li>
+</ul>
```

Comment:
Do you want anything from this list (you'll need to rewrite the titles from an end-user perspective)?

```
$ grep -E '^\* [a-z]+\(android' HISTORY.md | grep feat
* feat(android/engine): Add embedded KeymanWeb engine crash reporting to Sentry (#3825)
* feat(android/engine): Add check for associated model on ModelPickerActivity (#3808)
* feat(android): Use Stepper for navigating package installation steps (#3498)
* feat(android/app): Add language picker for keyboard package installation (#3481)
* feat(android): Use staging help site for pre-release builds (#3453)
* feat(android/app): Associate app with /keyboards/install links (#3449)
* feat(android): Use api.keyman-staging.com for pre-release builds (#3423)
* feat(android): Add app association to keyman.com (#3431)
* feat(android/app): Use build-download-resources.sh for KMApro app (#3322)
* feat(android/engine): Add additional info on installed keyboards exceptions (#3319)
* feat(android): Download default resources at build time (#3300)
* feat(android): Implement menu to add language for installed keyboard pkg (#3255)
* feat(android/app): Consolidate install menus (#3245)
* feat(android/engine): Migrate cloud keyboards when updating keyboard package (#3221)
* feat(android): Handle model picker updates (#3209)
* feat(android): Add utility to check BCP47 equivalence (#3210)
* feat(android): Add system globe action to show system keyboards (#3197)
* feat(android/samples): Add Tamil lexical model to KMSample2 (#3123)
* feat(android): Migrate installed keyboards list to keyboards_list.json (#3091)
* feat(android): Add KMPBrowserActivity for cloud keyboard searches (#2961)
* feat(android): Download cloud keyboards from https://keyman.com/keyboards (#2953)
* feat(android): Dismiss system keyboard on Back press (#2984)
* feat(android): Propagate languageID when downloading kmp (#2944)
* feat(android): Specify optional language ID for installing kmp (#2921)
* feat(android): Additional Sentry integration (#2810)
* feat(android): Start adding RTL to layouts (#2816)
* feat(android): Start of Sentry-based crash reporting (#2778)
* feat(android): Update additional main app strings for crowdin (#2793)
* feat(android): Handle keyman:// protocol to download kmp keyboard (#2734)
```